album/pokemon-sword-shield-super-music-collection.yaml

```diff
@@ -0,0 +1,1244 @@
+---
+Track: Battle Tower
+Directory: battle-tower-pokemon-sword-shield
+Duration: '1:48'
+Artists:
+- Minako Adachi
+URLs:
+- https://www.youtube.com/watch?v=NL95xW_8UxE
+Referenced Tracks:
+- Battle Tower (Johto) (Pokémon Crystal)
+---
+Track: Battle! (Battle Tower)
+Duration: '5:16'
+Artists:
+- Toby Fox
+URLs:
+- https://www.youtube.com/watch?v=shzaFbvGnhw
+Commentary: |- ##https://www.pokemon.com/us/pokemon-news/a-special-letter-and-song-from-undertale-game-creator-toby-fox
+    <i>Toby Fox:</i> ([A Special Letter and Song from Undertale Game Creator Toby Fox](https://www.pokemon.com/us/pokemon-news/a-special-letter-and-song-from-undertale-game-creator-toby-fox))
+    Hi everyone! I'm Toby Fox. Have you heard of me before? Well, that's ’cause I just said my name. LOL
+
+    Jokes aside, aren't you interested in how all of this happened? Well... OK. I'll tell you, I guess.
+
+    Once upon a time, I created an RPG game called UNDERTALE. For one reason or another, it seems a lot of people from GAME FREAK ended up becoming fans of this game.
+
+    One day, I noticed [[artist:go-ichinose|Go Ichinose-san]], one of the composers for Pokémon, was following me on Twitter, so... I messaged him out of the blue.
+
+    Eventually, we became friends and I was able to meet many other people from GAME FREAK, too! After eating sukiyaki with members of the company one day, we went to a cafe. There, I ordered a bottled green tea and selfishly, randomly asked if I could make a song for Pokémon someday...
+
+    ... they said yes! Right there in the cafe, they immediately told me a concept for a song I could make, and, as I walked home, my heart was burning with inspiration... a melody entered my head, and as I stumbled past salarymen in the dark, I sang it to myself over and over so I wouldn't forget it!
+
+    In the end, that song... became the song you can hear in the game!
+
+    It's a fierce track full of explosive power. I wanted to show the energy and passion I felt when asked to create a song for Pokémon! Yeahh! Let's get wild, baby!!! Ah, but wait, wait a second. It's the only song I made, and you can't hear it until you beat the game, so... make sure you enjoy GAME FREAK's music properly first, OK!? Adachi-san and Ichinose-san worked hard.... Don't rush through the whole game just to hear my song.... (No one would actually ever do that.)
+
+    Oh, and this goes without saying, but it's a huge honor to be allowed to make music for GAME FREAK! I'm so, so happy and thankful I was allowed to do this. So please look forward to the song, and to Pokémon Sword and Shield, as well! Yamper yam yam!
+
+    PS - If you like it, please let GAME FREAK know. They might treat me to sukiyaki again. (No one would ever actually do that.)
+
+    <i>Morris:</i> (wiki editor)
+    It doesn't seem to be an intentional reference, but people pointed out this song has a similar melody to [[track:a-baby-legend]].
```

Comment:
It's an intentional reference in my heart and in my imo we should list it as one